lib/parse/credit.js

```diff
@@ -4,6 +4,10 @@ import computeCredit from '../latepenalty'
 // TODO refactor/split this out
 
 export default function parseCredit(args) {
+  // unfinished, still in time promise
+  if (! args.finishDate && Date.now() < args.dueDate)
+    return NaN
```

Comment:
I think we should return `null` here, which is what Sequelize expects as an empty field value ( I should probably never have been using Nan really)

this way, our virtual credit db field will be `null` for pending promises, and [this](https://github.com/beeminder/iwill/pull/101/files#diff-3eccb1c372ca89fcfcd85532add0b383R26) check can simply be `if (credit) {`

Reply:
also, if this function returns `null` for pending promises, that's 99% of the work for #93 :D

Reply:
(just a side note for later: this is related to the rosycredit() and isPending() functions in the spec -- https://github.com/beeminder/iwill/wiki )